From 508f63c76e6c19bc02384941148358636ddb3b5c Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <jak@jak-linux.org>
Date: Sun, 26 Nov 2017 12:26:30 +0100
Subject: [PATCH] Define prefix in Makefile.autosetup

Debian tools and probably other tools too pass arguments like
	--mandir=\${prefix}/share/man
to configure. The prefix survives the output into the Makefile,
therefore, we need to define prefix in the Makefile too.
---
 Makefile.autosetup | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.autosetup b/Makefile.autosetup
index f54dbce1..eb8a0eb0 100644
--- a/Makefile.autosetup
+++ b/Makefile.autosetup
@@ -32,6 +32,7 @@ LIBBINFILES+=	$(SRCDIR)/contrib/smime_keys
 @endif
 
 # paths
+prefix=		@prefix@
 bindir=		@BINDIR@
 datadir=	@PKGDATADIR@
 docdir=		@PKGDOCDIR@
-- 
2.11.0

